client/my-sites/stats/activity-log/index.jsx

```diff
@@ -300,6 +308,47 @@ class ActivityLog extends Component {
 		);
 	}
 
+	renderUpgradeBanner() {
+		return (
+			<div className="activity-log__upgrade-banner">
+				{ this.props.isJetpackSite ? (
+					<Banner
+						callToAction="Get daily backups"
+						dismissPreferenceName="activity-upgrade-banner-jetpack"
+						event="track_event"
+						feature={ FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY }
+						list={ [
+							'See all site activity over the past month',
+							'Rewind your site back to any point',
+							'Automatic threat scanning',
+						] }
+						plan={ PLAN_JETPACK_PERSONAL }
+						title="Secure your site with daily backups"
+					/>
+				) : (
+					<div>
+						<FeatureExample>
+							<ActivityLogDemo />
+						</FeatureExample>
+						<Banner
+							callToAction="See all activity"
+							dismissPreferenceName="activity-upgrade-banner-simple"
+							event="track_event"
+							feature={ FEATURE_JETPACK_ESSENTIAL }
+							list={ [
+								'Changes made to your posts and pages',
+								'Historical record of site costumizations',
+								'Comments, pingbacks, and more',
+							] }
+							plan={ PLAN_PERSONAL }
+							title="See all your site activity"
+						/>
+					</div>
```

Comment:
We can work a little on the copy here. I would suggest:
"Upgrade your WordPress.com experience" for a title

For features, I'd list:
"Get a custom domain name"
"Remove WordPress.com ads"
"See 30 days of past activity"

And change the callToAction to "Upgrade"